server/index.js

```diff
@@ -9,7 +9,7 @@ const userClient = require('./data/userClient')
 const nomisClientBuilder = require('./data/nomisClientBuilder')
 const pdfFormatter = require('./services/utils/pdfFormatter')
 
-const notifyClient = new NotifyClient(config.notifications.notifyKey)
+const notifyClient = new NotifyClient(config.notificationConfig.notifyKey)
```

Comment:
not sure if `config.notificationConfig` is any better than just `config.notification`

Reply:
was to avoid name conflicts but your other suggestion fixes the problem ta